ServiceWebsite/ServiceWebsite.AcceptanceTests/Helpers/BookingsApiClientHelper.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Reflection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using RestSharp;
+using ServiceWebsite.AcceptanceTests.Configuration;
+using ServiceWebsite.AcceptanceTests.Models;
+using ServiceWebsite.AcceptanceTests.TestClients;
+using ServiceWebsite.BookingsAPI.Client;
+using ServiceWebsite.Configuration;
+
+namespace ServiceWebsite.AcceptanceTests.Helpers
+{
+    public class BookingsApiClientHelper
+    {
+        public BookingsApiTestClient CreateClientWithConfig(IConfigurationRoot configRoot)
```

Comment:
Maybe pass the just the string bookingApiUrl, string clientId...etc

Reply:
done